portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,32 +27,39 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 import java.util.ArrayList;
+import java.util.List;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
-    ArrayList<String> jsonData = new ArrayList<String>();
+    
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    int numOfComments;
 
     @Override
     public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {   
+        String numOfCommentsString = request.getParameter("numOfComments");
+        if (numOfCommentsString != null) {
```

Comment:
FYI, if the client makes a request such as `/data?numOfComment=`, then the parameter will be defined and the value will be an empty string (`""`). Usually, you would also check to make sure it's not an empty string.

Reply:
Done.